Source/Core/DolphinQt/MainWindow.cpp

```diff
@@ -163,6 +158,17 @@ void DMainWindow::StartGame(const QString filename)
 		}
 		m_render_widget->adjustSize();
 	}
+	else
+	{
+		m_render_widget->showMaximized();
```

Comment:
I have no idea, my window manager has no concept of maximized and unmaximized, so I assumed it did.
Do you want me to change it to a plain show()?